README.md

```diff
@@ -153,7 +157,7 @@ E.g. the string _(u_1 / into :1 (u_3 / pour))_ would describe a graph with two n
 evaluator.match_features(df, [[["(u_1 / into :1 (u_2 / pour) :2 (u_3 / YYY))"], [], "Entity-Destination(e1,e2)"]])
 ```
 
-Describing a subgraph with the string "(u_1 / into :1 (u_2 / pour) :2 (u_3 / YYY))" will return only three examples instead of 9, when we only had a single node as a feature
+Describing a subgraph with the string "(u_1 / into :1 (u_2 / pour) :2 (u_3 / YYY))" will return only three examples instead of 9 (when we only had a single node as a feature)
```

Comment:
Is there a reference to the penman format somewhere? How will users know how to read this?

Reply:
Yes, there is a note: "One of the core features of our tool is that we are also able to match subgraphs. To describe a graph, we use the PENMAN notation."